plugins/node/opentelemetry-instrumentation-express/README.md

```diff
@@ -45,7 +45,7 @@ registerInstrumentations({
 });
 ```
 
-See [examples/express](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-express/examples) for a short example.
+See [plugins/node/opentelemetry-instrumentation-express/examples](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-express/examples) for a short example.
```

Comment:
Since this README resides in the `plugins/node/opentelemetry-instrumentation-express` path, do we really need this long prefix?
IMO, simply writing `See [examples](...)` is clear enough, but I am open for other opinions

Reply:
you're right. I thought about that too but couldn't decide which way is better :)